lib/edition_assertions.rb

```diff
@@ -14,11 +14,25 @@ def initialize(edition, assertion)
     end
   end
 
+  class FeatureError < Error
+    def initialize(edition, assertion)
+      super(edition, assertion || "edition supports feature")
```

Comment:
This sounds positive but don't we want this to be negative as we raise this when the document type doesn't support a feature rather than it does? 

I notice the state one also seems to be positive so I'm not sure if I've missed something.

Reply:
Might help to point out these get interpolated into `"Assertion failed: #{assertion}"`.

Reply:
Oh yes, that does make sense.